test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -92,15 +103,25 @@ class test_ucp_ep_reconfig : public ucp_test {
         if (sender().ucph()->num_tls <= 2) {
             UCS_TEST_SKIP_R("test requires at least 2 ifaces to work");
         }
+
+        check_reused_lanes_reconfigurable();

```

Comment:
Is it really needed in every `init()`?

Reply:
how else will you check if reuse_lanes can be run?

Reply:
I mean - is it needed in every type of `test_ucp_ep_reconfig` (every test case in this file)?

Reply:
it is required for the base class (`test_ucp_ep_reconfig`).